app/assets/stylesheets/components/ui/_search_results.scss

```diff
@@ -21,3 +21,7 @@
   margin-top: $baseline-unit;
   text-decoration: underline;
 }
+
+.search-results__evidence-type {
+  margin: 1em 0;
```

Comment:
I think we should use $baseline-unit (or multiples thereof) to retain "vertical rhythm" here.

Reply:
This is going to be covered in a new ticket where a global reset will be applied